plugins/src/ios/OptimoveSDKPlugin.swift

```diff
@@ -34,4 +34,105 @@ import NotificationCenter
         guard let config = OptimoveSDKPlugin.config else { return }
         Optimove.initialize(with: config)
     }
+    
+    @objc(reportEvent:)
+    func reportEvent(command: CDVInvokedUrlCommand) {
+        var name = ""
+        var parameters: [String: Any] = [:]
```

Comment:
Should the default value for this be `nil`? (I think it's an optional arg to reportEvent)

Reply:
Ah cool (y)